examples/advanced/federated-statistics/df_stats/utils/prepare_data.py

```diff
@@ -13,11 +13,10 @@
 # limitations under the License.
 
 import argparse
+import csv
```

Comment:
do we need to add this one into "examples/advanced/federated-statistics/df_stats/requirements.txt"?
or it is builtin?